engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/content-hosts.html

```diff
@@ -51,6 +51,10 @@ <h2 translate>Content Hosts</h2>
           <a ng-click="openModuleStreamsModal()" disable-link="table.numSelected === 0" translate>Manage Module Streams</a>
         </li>
 
+        <li role="menuitem" ng-show="permitted('edit_hosts')" ng-class="{disabled: table.numSelected === 0}">
+          <a ng-click="openTracesModal()" disable-link="table.numSelected === 0" translate>Manage Host Traces</a>
```

Comment:
This should be the final functional change. In cases where REX is not installed I think this link should be disabled.

In the bulk traces modal controller you added this:

```
$scope.remoteExecutionPresent = BastionConfig.remoteExecutionPresent;
```

I think we actually need that in the controller for this view and to check for it here since there's no point in being able to get into the modal without REX, imo. What do you think?

With this, you can remove all of the 'REX' stuff from the bulk traces controller